internal/stackdriver/stackdriver.go

```diff
@@ -43,6 +44,45 @@ func NewAPIClient(ctx context.Context, project string) (*API, error) {
 	}, nil
 }
 
+// RequestCount count returns the number of requests for the given offset and
+// query.
+func (a *API) RequestCount(ctx context.Context, query Query, offset time.Duration) (int64, error) {
+	query = addFilterToQuery(query, "metric.type", requestCount)
+	endTime := time.Now()
+	startTime := endTime.Add(-1 * offset)
+	offsetString := fmt.Sprintf("%fs", offset.Seconds())
+
+	req := a.MetricClient.Projects.TimeSeries.List("projects/" + a.Project).
+		Filter(query.Query()).
+		IntervalStartTime(startTime.Format(time.RFC3339Nano)).
+		IntervalEndTime(endTime.Format(time.RFC3339Nano)).
+		AggregationAlignmentPeriod(offsetString).
+		AggregationPerSeriesAligner("ALIGN_DELTA").
+		AggregationGroupByFields("resource.labels.service_name").
+		AggregationCrossSeriesReducer("REDUCE_SUM")
+
+	resp, err := req.Do()
+	if err != nil {
+		return 0, errors.Wrap(err, "error when retrieving time series")
+	}
+	if len(resp.ExecutionErrors) != 0 {
+		return 0, errors.Errorf("execution errors occurred: %v", resp.ExecutionErrors)
```

Comment:
might be better to use %#v here OR actually loop over execution errors and log them as .warn

Reply:
Done. The tests will fail until #22 is merged.